test/metabase/db/liquibase_test.clj

```diff
@@ -110,7 +110,7 @@
               (is (= :done (liquibase/wait-for-all-locks sleep-ms timeout-ms))))))))))
 
 (deftest release-all-locks-if-needed!-test
-  (mt/test-drivers #{:h2 :mysql :postgres}
+  (mt/test-drivers #{:h2}
```

Comment:
Can we add a test for the other databases using the session lock?

Reply:
I don't think such a test would mean anything. You can't release the locks anymore from another process, you have to wait or kill the lock-holding process.

Reply:
I was thinking something like the following:

1. Start a thread which opens a connection with the lock
2. Have this thread then blocks on a latch.
2. Release the latch from the main thread, to let the thread exit and close its connection.
3. Confirm that you can connect.

A bit fiddly, since you'd have to bypass our connection pool, but without any tests we just have action of a distance when a dependency is added, and wouldn't catch a regression, even if the dependency was removed.